packages/tools/client-logger/app-insights-logger/package.json

```diff
@@ -0,0 +1,132 @@
+{
+	"name": "@fluid-tools/app-insights-logger",
+	"version": "2.0.0-internal.4.1.0",
+	"private": true,
+	"description": "Contains a fluid logging client that sends telemetry events to Azure App Insights",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/tools/client-logger/app-insights-logger"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
+		"build:compile": "npm run build:tsc && npm run build:copy-resources",
+		"build:copy-resources": "copyfiles \"src/**/*.css\" dist --up 1",
+		"build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"build:full": "concurrently npm:build npm:webpack",
+		"build:full:compile": "concurrently npm:build:compile npm:webpack",
+		"build:genver": "gen-version",
+		"build:tsc": "npm run tsc",
+		"ci:build:docs": "api-extractor run && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"clean": "rimraf _api-extractor-temp coverage dist nyc *.tsbuildinfo *.build.log",
+		"eslint": "eslint src",
+		"eslint:fix": "eslint src --fix",
+		"format": "npm run prettier:fix",
+		"format-and-build": "npm run format && npm run build",
+		"format-and-compile": "npm run format && npm run build:compile",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
+		"start:test-app": "start-server-and-test start:tinylicious 7070 start:test-app:client",
+		"start:test-app:client": "webpack serve --config webpack.config.js",
+		"start:tinylicious": "tinylicious",
+		"test": "npm run test:jest",
+		"test:coverage": "npm run test:jest:coverage",
+		"test:jest": "jest --detectOpenHandles",
+		"test:jest:coverage": "jest --detectOpenHandles --coverage --ci --reporters=default --reporters=jest-junite",
+		"tsc": "tsc",
+		"webpack": "webpack --env production",
+		"webpack:dev": "webpack --env development"
+	},
+	"dependencies": {
+		"@fluidframework/common-definitions": "^0.20.1",
+		"@fluidframework/test-driver-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
```

Comment:
this should not be reference outside of devDeps, test drivers are test only

Reply:
Removed in commit 70aa9fd